test/Microsoft.AspNet.Mvc.Core.Test/ActionResults/CreatedResultTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.PipelineCore.Collections;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc
+{
+    public class CreatedResultTests
+    {
+        [Fact]
+        public void CreatedResult_SetsLocation()
+        {
+            // Arrange
+            var location = "http://test/location";
+
+            // Act
+            var result = new CreatedResult(location, "testInput");
+
+            // Assert
+            Assert.Same(location, result.Location);
+        }
+
+        [Fact]
+        public async Task CreatedResult_ReturnsStatusCode_SetsLocationHeader()
+        {
+            // Arrange
+            var appRoot = "/home";
```

Comment:
what is this used for? I'd expect to see this in the output. 

If it's unused, then don't bother setting it.